src/components/view_file/view_file.vue

```diff
@@ -413,6 +499,7 @@ table {
 
 .view-file-component {
   overflow-y: auto;
+  position: relative;
```

Comment:
Adding this unfortunately messes up the context menu positioning, which is pretty important for handgrading. Is there another way to make the copy button appear in place? Or perhaps the copy button can just go next to the theme toggle switch?